utilities/storage.py

```
@@ -653,19 +653,20 @@ def data_volume_template_dict_with_pvc_source(
     size: str | None = None,
     storage_class: str | None = None,
 ) -> dict[str, Any]:
-    source_dv_pvc_spec = source_dv.pvc.instance.spec
+    source_dv.pvc.instance.spec
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)